.github/workflows/ci.yaml

```diff
@@ -71,6 +71,7 @@ jobs:
           --env BSD=${{matrix.bsd}}
           --env PYTESTFLAGS="--verbose -p no:cacheprovider"
           --env NETWORK=$NETWORK
+          --env LIVE_SSH_HOST=localhost
```

Comment:
Let's rename this envvar to `BASH_COMPLETION_TEST_LIVE_SSH_HOST`, pre-approved with that change.